src/app/services/submissions.service.ts

```diff
@@ -115,11 +115,10 @@ export class SubmissionsService {
       return response;
     } else {
       try {
-        const projectsLinks = activeSubmission['_links']['contents']['_links']['project']['href'];
-
-        this.http.get(projectsLinks).pipe(
+        const projectsLinks = activeSubmission['_links']['contents']['_links']['projects']['href'];
+        var projectResponse = this.http.get(projectsLinks).pipe(
           map(response => {
-            if (response.hasOwnProperty('_embedded') && response['_embedded'].hasOwnProperty('project')) {
+            if(response.hasOwnProperty("_embedded") && response['_embedded'].hasOwnProperty("projects")) {
```

Comment:
The replaced line is more lint friendly than the current one.

Reply:
Fixed in the new commmit